### ISSUE TITLE
Pretrained Model for Visual Genome

### ISSUE THREAD

**Issue by goyalsaransh97** [facebookresearch/Detectron#895]
Do you have/provide a pre-trained model for Mask-RCNN trained over Visual Genome dataset?
Or any script to directly train one, which much effort or modifications?

**Comment by Yudezhi** [comment]
Visual Genome can't used for instance segmentation
